.github/workflows/ci.yml

```diff
@@ -30,6 +31,16 @@ jobs:
           - 3.8
           - 3.9
     steps:
+      - name: Get PyTorch Channel
+        shell: bash
+        run: |
+          if [[ "${{ github.base_ref }}" == release/* ]] || [[ "${{ github.ref }}" == refs/heads/release/* ]] || [[ "${{ github.ref }}" == refs/tags/v* ]]; then
```

Comment:
Re-land due to `github.base_ref` can be an empty variable. Then, when it's empty, the bash command will fail.

Reply:
noob question: Does the last condition mean that if the branch has a tag that starts with 'v', it will choose `torch_test` as the channel?

Reply:
Yeah.

Reply:
Yep. It's one step during release. I haven't created one for 0.4.0 yet because I will create the first rc at the end of today to incorporate the timeline of cherry-pick. 
See all tags: https://github.com/pytorch/data/tags